### PR TITLE
Add proper linux instructions for Quark

### PR DESCRIPTION
**Pull request by KranKRival**
Hey Xor :D

### CODE REVIEW

**Comment by rkmax** [comment]
I think this instructions should be clear about the use of the oracle SDk instead the openJDK

File: README.md [hunk 1]
```diff
@@ -103,7 +109,23 @@ Only requirement for Quark to work is **JRE/JDK 8u60 or higher**.
...
+
+Only requirement for Quark to work is **JRE/JDK 8u60 or higher**.
+
+**One time setup**:
+
+1. Install OpenJDK or openJavaFX, both can be installed
+
+2. Open Goldleaf and connect your Switch via USB with your PC
+
+3. Open Terminal window at Quark.jar loacation and type "sudo java -jar Quark.jar" sudo is important
```
Comment (TehPsychedelic) [inline README.md]: location
and could rephrase as execute with superuser privileges

File: README.md [hunk 2]
```diff
@@ -103,7 +109,23 @@ Only requirement for Quark to work is **JRE/JDK 8u60 or higher**.
...
+
+4. Enter your Sudo password when promted and press enter
```
Comment (TehPsychedelic) [inline README.md]: prompted

**Comment by XorTroll** [comment]
Instructions will be fully remade for 0.8.

**Pull request closed without merging**